sessionctx/variable/session.go

```diff
@@ -982,6 +988,19 @@ const (
 	SlowLogMemMax = "Mem_max"
 )
 
+// BackoffTimer contains variable back off time.
+type BackoffTimer struct {
+	waitScatterRegionFinish int
+}
+
+// GetWaitScatterRegionFinishBackoff gets the back off time of waitScatterRegionFinish.
+func (b *BackoffTimer) GetWaitScatterRegionFinishBackoff() int {
+	if b.waitScatterRegionFinish > 0 {
```

Comment:
If we use `tidbOptPositiveInt32` to set the value. I think we needn't do this.

Reply:
done. Thanks.